integration/kubernetes/k8s-liveness-probes.bats

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bats
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+load "${BATS_TEST_DIRNAME}/../../.ci/lib.sh"
+
+setup() {
+	export KUBECONFIG=/etc/kubernetes/admin.conf
+	pod_name="liveness-exec"
+}
+
+@test "Liveness probe" {
+	wait_time=120
+	sleep_time=5
+	sleep_liveness=7
```

Comment:
iiuc, this 7 correlates to the 'sleep 6' in the busybox image - having the extra 1s to ensure the pod has finished. I think just a 1s gap is pushing our luck, and it won't surprise me if one day we find this test sometimes passes. Maybe bump it a little to get more of a guarantee the pod has quit?